Api/RecommendManagementInterface.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace Algolia\AlgoliaSearch\Api;
```

Comment:
Thanks for your feedback. I respectfully disagree with this. I think it would be benefited if we had a lots of interfaces under the Api directory. This is the 2nd interface in this directory. To me it make sense to keep it here.

Reply:
Not sure I understand this objection. We actually have many interfaces under `Api` (more than 2) but they are currently grouped by subdirectory. My suggestion was to try and keep things better organized. We only provide recommendations for products so that fits in my mind. I suppose we could keep it at the root level if you wish. What is the benefit to doing it this way?

Reply:
Sorry for the confusion, I was talking about interfaces in the root Api directory. There is only one interface in there now. Most ManagementInterfaces are located in the root Api directory. I prefer to follow the pattern. To me that's the default place. We should ask this, what's the benefit of moving it to product directory? :). We can try organizing it but to me it's not unorganized as it is now.

Reply:
What about creating an `Algolia\AlgoliaSearch\Api\Recommend` subdirectory ?